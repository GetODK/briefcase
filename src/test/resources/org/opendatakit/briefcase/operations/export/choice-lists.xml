<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml">
  <h:head>
    <h:title>Choice lists</h:title>
    <model>
      <instance>
        <data id="choice-lists">
          <some_string/>
          <another_string/>
          <some_group>
            <yet_another_string/>
          </some_group>
          <meta>
            <instanceID/>
          </meta>
        </data>
      </instance>
      <bind nodeset="/data/some_string" type="string"/>
      <bind nodeset="/data/another_string" type="string"/>
      <bind nodeset="/data/some_group/yet_another_string" type="string"/>
      <bind calculate="concat('uuid:', uuid())" nodeset="/data/meta/instanceID" readonly="true()" type="string"/>
    </model>
  </h:head>
  <h:body>
    <select ref="/data/some_string">
      <item>
        <value>CHOICE_1</value>
        <label>CHOICE 1</label>
      </item>
      <item>
        <value>CHOICE_2</value>
        <label>CHOICE 2</label>
      </item>
    </select>
    <select1 ref="/data/another_string">
      <item>
        <value>CHOICE_1</value>
        <label>CHOICE 1</label>
      </item>
      <item>
        <value>CHOICE_2</value>
        <label>CHOICE 2</label>
      </item>
    </select1>
    <group>
      <label>Some group</label>
      <select ref="/data/some_group/yet_another_string">
        <item>
          <value>CHOICE_1</value>
          <label>CHOICE 1</label>
        </item>
        <item>
          <value>CHOICE_2</value>
          <label>CHOICE 2</label>
        </item>
      </select>
    </group>
  </h:body>
</h:html>
