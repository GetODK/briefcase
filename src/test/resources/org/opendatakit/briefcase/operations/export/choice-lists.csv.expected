SubmissionDate,some_string,some_string/CHOICE_1,some_string/CHOICE_2,another_string,some_group-yet_another_string,some_group-yet_another_string/CHOICE_1,some_group-yet_another_string/CHOICE_2,meta-instanceID,KEY
"Apr 26, 2018 8:58:20 AM",CHOICE_1,1,0,CHOICE_2,CHOICE_1,1,0,uuid:39f3dd36-161e-45cb-a1a4-395831d253a7,uuid:39f3dd36-161e-45cb-a1a4-395831d253a7
